azureproject/production.py

```diff
@@ -1,21 +1,19 @@
 import os
+import secrets
 
 # SECURITY WARNING: keep the secret key used in production secret!
-# Use this py command to create secret 
-# python -c 'import secrets; print(secrets.token_hex())'
-SECRET_KEY = os.getenv('AZURE_SECRET_KEY')
+SECRET_KEY = os.environ['SECRET_KEY'] if 'SECRET_KEY' in os.environ else secrets.token_hex()
```

Comment:
```suggestion
SECRET_KEY = os.getenv('SECRET_KEY') or secrets.token_hex()
```

You could also do os.getenv('SECRET_KEY', secrets_token_hex()) but I prefer the code above because environment variables could exist but be empty strings, in which case they may as well not exist!